nix/container/libvirtd_base.nix

```diff
@@ -0,0 +1,180 @@
+{ pkgs
+, pkgsLinux
+, stdenv
+}:
+let
+  passwd = pkgs.writeTextDir "etc/passwd" ''
+    root:x:0:0:root:/root:/bin/sh
+    bin:x:1:1:bin:/bin:/sbin/nologin
+    daemon:x:2:2:daemon:/sbin:/sbin/nologin
+    adm:x:3:4:adm:/var/adm:/sbin/nologin
+    lp:x:4:7:lp:/var/spool/lpd:/sbin/nologin
+    sync:x:5:0:sync:/sbin:/bin/sync
+    shutdown:x:6:0:shutdown:/sbin:/sbin/shutdown
+    halt:x:7:0:halt:/sbin:/sbin/halt
+    mail:x:8:12:mail:/var/spool/mail:/sbin/nologin
+    operator:x:11:0:operator:/root:/sbin/nologin
+    games:x:12:100:games:/usr/games:/sbin/nologin
+    ftp:x:14:50:FTP User:/var/ftp:/sbin/nologin
+    nobody:x:65534:65534:Kernel Overflow User:/:/sbin/nologin
+    tss:x:59:59:Account used for TPM access:/:/usr/sbin/nologin
+    dbus:x:81:81:System message bus:/:/sbin/nologin
+    unbound:x:999:999:Unbound DNS resolver:/var/lib/unbound:/sbin/nologin
+    saslauth:x:998:76:Saslauthd user:/run/saslauthd:/sbin/nologin
+    systemd-network:x:192:192:systemd Network Management:/:/usr/sbin/nologin
+    systemd-oom:x:997:997:systemd Userspace OOM Killer:/:/usr/sbin/nologin
+    systemd-resolve:x:193:193:systemd Resolver:/:/usr/sbin/nologin
+    polkitd:x:996:996:User for polkitd:/:/sbin/nologin
+    rtkit:x:172:172:RealtimeKit:/proc:/sbin/nologin
+    gluster:x:995:995:GlusterFS daemons:/run/gluster:/sbin/nologin
+    dnsmasq:x:994:994:Dnsmasq DHCP and DNS server:/var/lib/dnsmasq:/usr/sbin/nologin
+    rpc:x:32:32:Rpcbind Daemon:/var/lib/rpcbind:/sbin/nologin
+    rpcuser:x:29:29:RPC Service User:/var/lib/nfs:/sbin/nologin
+    qemu:x:107:107:qemu user:/:/sbin/nologin
+    pipewire:x:993:992:PipeWire System Daemon:/var/run/pipewire:/sbin/nologin
```

Comment:
I see that this is only moved, but maybe we can clean this up eventually? :D

Reply:
Cleaned up unneded users / groups:

https://github.com/edgelesssys/constellation/blob/b424a15cb7f4ad8b63ad677bfea72900a6dc20bf/nix/container/libvirtd_base.nix#L6-L52